instances/treasury-devdao.near/widget/pages/dashboard/TransactionHistory.jsx

```diff
@@ -0,0 +1,384 @@
+const { readableDate } = VM.require(
+  "${REPL_DEVHUB}/widget/core.lib.common"
+) || { readableDate: () => {} };
+
+const { nearPrice } = props;
+const [transactionWithBalances, setTransactionWithBalance] = useState(null);
+const [page, setPage] = useState(1);
+const [showMoreLoading, setShowMoreLoading] = useState(false);
+const [hideViewMore, setHideViewMore] = useState(false);
+const totalTxnsPerPage = 20;
+const code = `
+<!doctype html>
+<html>
+  <head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
```

Comment:
You only need the script tag here, since this iframe will not display anything, right?